manifests/00-cluster-role.yaml

```diff
@@ -155,6 +155,7 @@ rules:
   - networking.k8s.io
   resources:
   - ingressclasses
+  - networkpolicies
   verbs:
   - '*'
```

Comment:
If possible, avoid adding wildcard rules.  We have complaints about existing wildcards in RBAC rules, and we should avoid adding more.